src/dataIntegration/transcriptomics/FASTCORE/fastcore.m

```diff
@@ -1,65 +1,51 @@
 % the minimum of reactions possible to support the core.
 %
 % USAGE:
 %
-%    tissueModel = fastcore(core, model, epsilon, expressionRxns, threshold, printlevel)
+%    tissueModel = fastcore(model,core)
 %
 % INPUTS:
-%    core:              indices of reactions in cobra model that are part of the
-%                       core set of reactions (called `C` in `Vlassis et al,
-%                       2014`)
-%    model:             input model (COBRA model structure)
+%
+%   model:               input model (COBRA model structure)   
+%   core:                indices of reactions in cobra model that are part of the
+%                        core set of reactions (called 'C' in 'Vlassis et al,
+%                        2014')
 %
 % OPTIONAL INPUTS:
-%    epsilon:           smallest flux value that is considered nonzero
-%                       (default 1e-8)
-%    expressionRxns:    expression data, corresponding to model.rxns (see
-%                       mapGeneToRxn.m)
-%    threshold:         expression threshold (reactions with expression
-%                       above this threshold are put in the set of core
-%                       reactions
-%    printLevel:        0 = silent, 1 = summary, 2 = debug
+%   epsilon:             smallest flux value that is considered nonzero
+%                        (default 1e-8)               
+%   printLevel:          0 = silent, 1 = summary, 2 = debug
+%
+% OUTPUTS:
 %
-% OUTPUT:
 %   tissueModel:         extracted model
 %
 %
-% `Vlassis, Pacheco, Sauter (2014). Fast reconstruction of compact
-% context-specific metbolic network models. PLoS Comput. Biol. 10, e1003424.`
+% 'Vlassis, Pacheco, Sauter (2014). Fast reconstruction of compact
+% context-specific metbolic network models. PLoS Comput. Biol. 10, e1003424.'
 %
 % .. Authors:
 %       - Nikos Vlassis, Maria Pires Pacheco, Thomas Sauter, 2013 LCSB / LSRU, University of Luxembourg
 %       - Ronan Fleming, commenting of code and inputs/outputs
 %       - Anne Richelle, code adaptation to fit with createTissueSpecificModel
 
 
-    if ~exist('printLevel','var')
-        %For Compatability with the original fastcore syntax
-        printlevel = 1;
-    end
-
-    %Define the set of core reactions
-    if ~isempty(expressionRxns) && ~isempty(threshold)
-        %additional option to extend the core set of reaction depending on
-        %a threshold on the gene expression data
-        coreSetRxn = find(expressionRxns >= threshold);
-        coreSetRxn= union(coreSetRxn, find(ismember(model.rxns, core)));
-    else
-        coreSetRxn = core;
+    if nargin < 4 || ~exist('printLevel','var')
+        printlevel = 0;
     end
-
-    if ~isempty(epsilon)
-        epsilon=1e-8;
+    if nargin < 3 || isempty(epsilon)
+        epsilon=1e-4;
     end
 
+    coreSetRxn = core;    
     model_orig = model;
 
     %Find irreversible reactions
-    irrevRxns = find(model.lb>=0); %% could be called irrevRxns
+    irrevRxns = find(model.rev==0);
```

Comment:
@arichelle, I think that @tpfau deprecated `model.rev` recently in PR #507. Also, in PR #513, @tpfau already had fixed this specific line. 

May you revert back to what he had changed? Thanks.